src/Find.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Find (findMatches, recipeToString) where
+
+import Types
+import RichText (toStr)
+import Text.Regex.TDFA ((=~))
+
+findMatches :: String -> [Recipe] -> [String]
+findMatches rgx = concatMap zipName 
+    where zipName r = 
+            let name = recipeName r 
+             in zipWith (\a b -> a ++ ": " ++ b) (repeat name) . findInRecipe rgx $ r
```

Comment:
You could simplify this line (note - I haven't actually tested this, but I _think_ it's equivalent):
```suggestion
            in map (\b -> name ++ ": " ++ b) (findInRecipe rgx r)
```

Reply:
Fixed, thank you for the suggestion!